apollo-router/src/configuration/mod.rs

```diff
@@ -710,7 +710,10 @@ impl Supergraph {
 
 impl Default for Supergraph {
     fn default() -> Self {
-        Self::builder().build()
+        Self::builder()
+            .generate_query_fragments(false)
+            .reuse_query_fragments(true)
```

Comment:
@BrynCooke I didn't mean to push this, but you approved. Based on what you said, I don't think this is what we wanted. Shall I remove it, or is this actually correct?

Reply:
`reuse_query_fragments` is still an `Option<bool>` though (you had suggested I change it to a `bool` in a separate thread, but I wasn't certain that's appropriate - maybe it is, but it seemed that way for a reason), so I think returning just `reuse_query_fragments` in this case is correct.

Reply:
Fair enough. Also I think I may have been mixing up `generate_query_fragments` and `reuse_query_fragments` in the statements above.